File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/RiskyTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class RiskyTag extends Tag {
    public RiskyTag(int count) {
        super(count, Constants.RISKY_TAG, Constants.RISKY_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/RiskyTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class RiskyTag(count: Int) : Tag(count, Constants.RISKY_TAG, Constants.RISKY_TAG_REMOTE, HashSet(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))